pkg/util/rangeio/rangeio.go

```diff
@@ -355,3 +412,17 @@ func optimizeRanges(cfg *Config, in []Range) []Range {
 	}
 	return out
 }
+
+type bytesStringer uint64
+
+func (s bytesStringer) String() string {
+	return humanize.Bytes(uint64(s))
+}
```

Comment:
Maybe I'm over-optimizing here but since we can call ReadRanges quite often, I don't want to allocate a string if the trace isn't being recorded, so I'm using a stringer impl to avoid that.